bblfsh/client.py

```diff
@@ -24,13 +24,14 @@ def __init__(self, endpoint):
                          for example "0.0.0.0:9432"
         :type endpoint: str
         """
+        Node.initialize()
```

Comment:
I don't think a libuast `Node` should be initialized in a `GRPC` client, why is it this way?